lua/nvim-autopairs/fastwrap.lua

```diff
@@ -59,7 +59,7 @@ M.show = function(line)
         end
         local list_pos = {}
         local index = 1
-        local str_length = #line
+        local str_length = #line + 1
```

Comment:
That change is suspicious, the length of the line is `#line`, if you add 1, it cannot be named `str_length` anymore, it's something else 🤔

Reply:
This doesn't interact well with `manual_position`, which isn't documented and doesn't have tests either. Something is going to break.